### ISSUE TITLE
Type inference for empty multiset type parameters doesn't work inside macros

### ISSUE THREAD

**Issue by fpoli** [fpoli/viper-silver#117]
```scala
define bar Multiset()

method foo(b : Bool) {
  assert Multiset(null) == bar
}
```

This code causes an error concerning an unknown type #E.

[Issue created by alexander_summers: 2019-07-07]
[Last updated on bitbucket: 2019-09-30]

[Comment created by alexander_summers: 2019-07-18]
Duplicate of #282.

[Comment created by fabiopakk: 2019-09-30]
Solved its duplicate #282.

**Comment by fpoli** [comment]
> **@mueller55** commented on 2016-02-08 16:34

**Comment by fpoli** [comment]
> **@meilers** commented on 2016-03-09 10:08

Fixed in commit c4cc18f.
